File: backend/src/main/java/br/ufpr/sept/so2/modules/iam/application/IamSettings.java
package br.ufpr.sept.so2.modules.iam.application;

public interface IamSettings {

    long accessTtlSeconds();

    long refreshTtlSeconds();

    long resetTtlSeconds();

    int maxFalhasConsecutivas();

    int minutosBloqueio();

    String frontendBaseUrl();
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/modules/iam/application/IamSettings.kt
package br.ufpr.sept.so2.modules.iam.application

interface IamSettings {
    val accessTtlSeconds: Long

    val refreshTtlSeconds: Long

    val resetTtlSeconds: Long

    val maxFalhasConsecutivas: Int

    val minutosBloqueio: Int

    val frontendBaseUrl: String
}
